doc-content/enterprise-only/bpmn/bpmn-user-task-permission-ref.adoc

```diff
@@ -0,0 +1,93 @@
+[id='bpmn-user-task-permission-matrix-ref']
+= BPMN2 task permission matrix in process designer
+
+The user task permission matrix summarizes the actions that are allowed for specific user roles. The user roles are as follows:
+
+* Potential owner: User who can claim the task, when the task is already claimed and has been released and forwarded. The tasks with `Ready` status can be claimed, and the potential owner becomes the actual owner of the task.
```

Comment:
I don't understand this sentence:
`User who can claim the task, when the task is already claimed and has been released and forwarded.`
How can a user claim the task when the task is already claimed? Probably you mean that the task has been claimed in the past and since then has been released and forwarded?

Reply:
Yes, that's correct, the task was claimed and has been released and forwarded. I can change the wordings with better clarity. 

Thanks Marian

Reply:
@MarianMacik, Have updated the sentence. 
Thanks for your input.